gulpfile.js

```diff
@@ -69,6 +70,37 @@ var webpackConfig = {
   plugins: gulpPlugins
 };
 
+gulp.task('test-unit', function (done) {
+    new KarmaServer({
+        configFile: __dirname + '/karma.conf.js'
+    }, done).start();
```

Comment:
I've always found it nice to add two tasks; One task which has `singleRun: true`, and another with `singleRun: false`. This allows for one task to be used for CI, ie automatically kicked off from Travis, and another for local development purposes.